.nycrc.json

```
@@ -4,11 +4,6 @@
     "text",
     "text-summary"
   ],
-  "exclude": [
-    "**/lib/*.js",
-    "**/*.css*",
-    "test"
-  ],
   "check-coverage": true,
   "lines": 72.5,
   "branches": 100,
```

Let's lower branch coverage to 81.5 for now
```suggestion
  "branches": 81.5,
```